exp/api/v1beta1/gcpmanagedcontrolplane_types.go

```diff
@@ -184,7 +191,13 @@ type GCPManagedControlPlaneStatus struct {
 
 	// CurrentVersion shows the current version of the GKE control plane.
 	// +optional
+	//
+	// Deprecated: This field will soon be removed and you are expected to use Version instead.
 	CurrentVersion string `json:"currentVersion,omitempty"`
+
+	// Version represents the version of the GKE control plane.
+	// +optional
+	Version *string `json:"version,omitempty"`
 }
```

Comment:
We want to populate this `status.Version` with whatever we currently populate the `status.CurrentVersion` with.
Then When in the future we remove the `status.CurrentVersion`, we will only have the `status.Version` to set.

One instance I saw of this is here: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/d80bfabadd3f611f5282d3ee7c2e963f3ffc4621/cloud/services/container/clusters/reconcile.go#L99

But there may be others.